src/SFA.DAS.EmployerIncentives.Web/ViewModels/Apply/SelectApprenticeships/SelectApprenticeshipsRequest.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Collections.Generic;
+
+namespace SFA.DAS.EmployerIncentives.Web.ViewModels.Apply.SelectApprenticeships
+{
+    public class SelectApprenticeshipsRequest : ISelectedApprenticeships
+    {
+        public string AccountId { get; set; }
+
+        public string AccountLegalEntityId { get; set; }
+
+        public List<string> SelectedApprenticeships { get; set; } = new List<string>();
+
+        public bool HasSelectedApprenticeships => SelectedApprenticeships.Count > 0;
```

Comment:
is it quicker to use Any()

Reply:
no, not for `Lists`, see here: https://stackoverflow.com/a/305156

cc: @David-B-Read